Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -284,28 +305,52 @@ private String applyRegex(String text) {
         }
         return text;
     }
-    
+
     private static Boolean IsMocked() {
-        // TODO: support un mocked mode
+        String keyValue=System.getProperty("isMockedCase");
```

Comment:
Yes. Does this mean I shouldn't use this method?  I should use a new method like "isMockedTestCase" ?